PhpUnit/DefinitionIsChildOfConstraint.php

```diff
@@ -46,7 +47,13 @@ public function toString()
 
     private function evaluateDefinitionIsDecorator(Definition $definition, $returnResult)
     {
-        if (!($definition instanceof DefinitionDecorator)) {
+        if (!$definition instanceof ChildDefinition && !$definition instance of DefinitionDecorator) {
```

Comment:
`instanceof` instead of `instance of` :)

Reply:
Sorry. I need to use my glasses..